azurelinuxagent/common/event.py

```diff
@@ -53,15 +58,13 @@
 
 MAX_NUMBER_OF_EVENTS = 1000
 
+EVENT_FILE_EXTENSION = '.waagent.tld'
+EVENT_FILE_REGEX = re.compile(r'(?P<agent_event>\.waagent)?\.tld$')
```

Comment:
Same, rename this to `AGENT_EVENT_FILE_REGEX` for clarity

Reply:
this one is different: it matches all event files (new agent, old agent and extensions)